### DISCUSSION TITLE
可不可以出个视频教程教一下配置所有的左边和右边的功能，尤其是新的WordPress版本，菜单、提交日期的小方块等

### DISCUSSION THREAD

**Discussion by ysn2382617094wsw** [Tokinx/Wing#143] [问答]
可不可以出个视频教程教一下配置所有的左边和右边的功能，尤其是新的WordPress版本，菜单、提交日期的小方块等，这个主题非常好看和简洁，简约但是不失大气，但是我不会配置就很难受，希望可以出个教程，视频教程最好！

**Comment by Tokinx** [comment]
这个建议B站或者Youtube找下视频学习一下WordPress哈。
